cmake/external/googleapis.cmake

```diff
@@ -103,7 +113,6 @@ if (NOT TARGET googleapis_project)
                    -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS}
                    -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
                    -DCMAKE_INSTALL_RPATH=${GOOGLE_CLOUD_CPP_INSTALL_RPATH}
-                   -DCMAKE_SKIP_INSTALL_RPATH=ON
```

Comment:
Is there any particular reason why this is removed?

Reply:
I cannot recall. Should we add it back?

Reply:
Yes. It was added because of https://github.com/googleapis/google-cloud-cpp/issues/2286. rpmbuild doesn't like an rpath of "external/lib" because it's "insecure", but it doesn't seem worth the effort to fix the RPATH unless somebody runs into a weird corner case. So the simplest solution is just to skip it.